sycl/test-e2e/SubGroupMask/GroupSize.cpp

```diff
@@ -15,8 +15,13 @@
 //
 //===----------------------------------------------------------------------===//
 
+// TODO: switch to <sycl/detail/core.hpp> once we decide how feature macros
```

Comment:
Do we need this TODO? It looks like test is using core.hpp now.

Reply:
You're right. I'll remove that in a follow-up PR because I want this merged to unblock dependent changes.

Reply:
Changing it as part of https://github.com/intel/llvm/pull/13853.